lib/widgets/layouts/adaptive_layout/adaptive_scaffold_view.dart

```diff
@@ -196,25 +189,17 @@ class AppScaffoldView extends StatelessWidget {
   }
 
   List<NavigationDestination> getNavigationDestinations(BuildContext context) {
-    return [
-      NavigationDestination(
-        icon: const TwakeNavigationIcon(
-          icon: Icons.contacts_outlined,
-        ),
-        label: L10n.of(context)!.contacts,
-      ),
-      NavigationDestination(
-        icon: UnreadRoomsBadge(
-          filter: (room) => !room.isSpace && !room.isStoryRoom,
-        ),
-        label: L10n.of(context)!.chat,
-      ),
-      NavigationDestination(
-        icon: const TwakeNavigationIcon(
-          icon: Icons.web_stories_outlined,
-        ),
-        label: L10n.of(context)!.stories,
-      ),
-    ];
+    return destinations.map((destination) {
```

Comment:
I think it's not good
in the future i want add story tab with badge, how can I do that
just change the destination.getTab extension